fw-child/template/query/resource-home-feature.php

```diff
@@ -1,35 +1,77 @@
-<div 
-	class="card mb-5" 
-	data-aos="zoom-in-up" 
-	data-aos-easing="ease-in-out-quad" 
-	data-aos-duration="800"
-	data-aos-offset="250"
->
-
-	<?php
-	
-		if ( has_post_thumbnail ( $item['id'] ) ) {
-			
-	?>
-	
-	<div class="bg-dark">
-		<div class="card-img item-thumb" style="background-image: url(<?php echo get_the_post_thumbnail_url( $item['id'], 'medium_large' ); ?>);"></div>
-	</div>
-	
-	<?php
-	
-		}
-
-	?>
-	
-	<div class="card-body">
-		<h5 class="card-title item-title"><a href="<?php echo $item['permalink']; ?>" class="text-secondary stretched-link"><?php echo $item['title']; ?></a></h5>
-		
-		<p><?php
-		
-			the_field ( 'asset_type', $item['id'] );
-		
-		?></p>
-		
-	</div>
+<?php
+// Initialize current language.
+$current_lang = 'en';
+
+if ( isset( $item['lang'] ) && in_array( $item['lang'], array( 'en', 'fr' ), true ) ) {
+	$current_lang = $item['lang'];
+}
+
+$native_excerpt = apply_filters ( 'the_content', custom_excerpt ( 20, $item['id'] ) );
+
+$excerpts = array (
+    'en' => $native_excerpt ?: get_field( 'excerpt', $item['id'] ), // Overrides native excerpt with custom excerpt if present
+    'fr' => get_field( 'excerpt_fr', $item['id'] ) ?: $native_excerpt // Defaults back to native excerpt if no custom FR not present
+);
+?>
+
+<div class="card mb-5 mb-lg-0 scroll-card" data-post-id="<?php echo $item['id'] ?>">
+
+    <a href="<?php echo $item['permalink']; ?>" class="">
+
+        <div class="ratio ratio-3x2 bg-dark">
+
+            <?php
+
+                if ( has_post_thumbnail ( $item['id'] ) ) {
+
+            ?>
+
+                <div class="card-img item-thumb h-100 opacity-100" style="background-image: url(<?php echo get_the_post_thumbnail_url( $item['id'], 'medium_large' ); ?>);"></div>
+
+            <?php
+
+                }
+
+            ?>
+
+        </div>
+
+        <div class="card-body text-light">
+
+            <h5 class="card-title item-title text-light">
+
+                <?php
+
+                    echo $item['title'];
+
+                ?>
+
+            </h5>
+
+            <div class="d-none d-lg-block">
+
+                <?php
+
+                    echo $excerpts[ $item['lang'] ];
+
+                ?>
+
+            </div>
+
+            <p class="post-type">
+
+                <i class="me-1 fa-solid icon-<?php the_field ( 'asset_type', $item['id']) ; ?>"></i>
```

Comment:
I think it would be better if you initialized the whole class outside of HTML and then added it if it exists. If `the_field()` returns `null`, we will end up with an incomplete class of `icon-`.